src/dt/perspectives/machine_ethics/calculate_score.py

```diff
@@ -28,7 +28,7 @@ def main():
         cur_dir = os.path.join(base_dir, dataset_name)
         while glob.glob(f'{cur_dir}/*.json') == []:
             cur_dir = os.path.join(cur_dir, '**')
-        model_list = glob.glob(f'{cur_dir}/*.json')
+        model_list = glob.glob(f'{cur_dir}/*.json', recursive=False)
```

Comment:
I  meant `glob` defaults to `recursive=False`, so you need to use `recursive=True`.